### PR TITLE
Module-1: Project Review From the Collaborative Trials

### PR DESCRIPTION
**Pull request by amare1990**
- Copy code from the previous collaborative trials project and add in this project
- Add acknowledgement in README.md file

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+
+@media (max-width:1000px) {
+  body {
+      background-color: rgb(120, 179, 120);
+  }
+
+}
+
+#navbar {
+font-family:Verdana, Geneva, Tahoma, sans-serif;
```
Comment (amare1990) [inline css/styles.css:21]: Please try to stick to the template you used. You are required to develop a project that looks like the template. You used a font-family (font-family:Verdana, Geneva, Tahoma, sans-serif) different from the font-family the template uses. The template uses Poppins as a font-family for most of the texts in the document. Therefore, put your font-family inside body CSS selector. Using one line of code makes your code compact and easy to modify.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,496 @@
...
+
+    - The freeCodeCamp Team
+
+    -->
+  </head>
+
+  <body>
+    <main id="main-doc">
+      <section class="main-section" id="Introduction">
+        <header>Introduction</header>
```
Comment (amare1990) [inline index.html:24]: Kindly replace header selector with hx, x ranging from 2-6. In one page, text embedded with in h1 tag is used once in a page. Header CSS selector is used to include headers (logo and menu).

File: index.html [hunk 3]
```diff
@@ -0,0 +1,496 @@
...
+    </main>
+
+    <nav id="navbar">
+      <header>JS Documentation</header>
+      <ul>
+        <li><a class="nav-link" href="#Introduction"> Introduction</a></li>
+        <li>
+          <a class="nav-link" href="#What_you_should_already_know"
+            >WHAT YOU SHOULD ALREADY KNOW</a
+          >
```
Comment (amare1990) [inline index.html:457]: Kindly move this closing symbol to a tag. You have to follow the normal HTML writing style. The opening and closing symbol (<, >) should be sticked together with the tags.

File: js/index.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+    (should all be failing at first)
+  - Start coding! As you fulfill each
+    test case, you will see them go   
+    from red to green.
+  - As you start to build out your 
+    project, when tests are failing, 
+    you should get helpful errors 
+    along the way!
+    ************/
+
```
Comment (amare1990) [inline js/index.js:24]: Kindly remove this file. It is unnecessary as this project does not require JavaScript code.

File: babel/index.babel [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+    from red to green.
+  - As you start to build out your 
+    project, when tests are failing, 
+    you should get helpful errors 
+    along the way!
+    ************/
+
+// PLEASE NOTE: Adding global style rules using the * selector, or by adding rules to body {..} or html {..}, or to all elements within body or html, i.e. h1 {..}, has the potential to pollute the test suite's CSS. Try adding: * { color: red }, for a quick example!
+
+// Once you have read the above messages, you can delete all comments.
```
Comment (amare1990) [inline babel/index.babel:27]: Kindly remove this file. Now you are using code reviewer using pull requests rather than testing it from codepin.

**Review by amare1990 — COMMENTED** [review]
Hi @amare1990. You are almost done . However, some tasks remain unsolved.

Changes required

Kindly check my comments below and act accordingly.

-Submit for review when you complete the required changes.
Thanks

File: css/styles.css (hunk 1 above)
Comment (ichala) [reply css/styles.css:21]: @amare1990 kindly try to include an explanation "why" for your feedback, to help the student understand why you requesting the changes

File: index.html (hunk 2 above)
Comment (ichala) [reply index.html:24]: @amare1990 kindly try to include an explanation "why" for your feedback, to help the student understand why you requesting the changes

File: index.html (hunk 3 above)
Comment (ichala) [reply index.html:457]: @amare1990 kindly try to include an explanation "why" for your feedback, to help the student understand why you requesting the changes

**Review by ichala — CHANGES_REQUESTED** [review]
Hi @amare1990 , 

Good job so far! 👏👏👏 
There are some issues that you still need to work on to go to the next project but you're almost there!

the-simpsons-homer-simpson

## To Highlight 🎉

-  Good use of github features ✅ 

 ## Required Changes ♻️

- [ ] **_Check the comments below_**

## Optional suggestions

N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by frank1738 — APPROVED** [review]
Hi @amare1990 , 

Good work implementing the changes requested. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

YesYouAreGIF (2)


## Highlights

✅ Correct use of Github Inline comments
✅ Provide a reason for requesting a change

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by amare1990** [comment]
Hi, @ichala.

I have added 5 feedbacks/comments. I used inline Github inline comment
features to give feedback.

**Pull request merged**